app/models/course/condition.rb

```diff
@@ -3,4 +3,6 @@ class Course::Condition < ActiveRecord::Base
 
   belongs_to :course, inverse_of: false
   belongs_to :conditional, polymorphic: true
+
+  delegate :satisfied_by?, to: :actable
```

Comment:
Okay, then the generic acts_as_condition cannot hardcode Course::Condition. Can you parameterise it?